src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/TransactionService.java

```diff
@@ -30,9 +30,13 @@ public Transaction getTransaction(String transactionId, String passthroughHeader
 
     public void updateTransaction(Transaction transaction, String passthroughHeader) throws ServiceException {
         try {
-            var uri = "/transactions/" + transaction.getId();
-            apiClientService.getOauthAuthenticatedClient(passthroughHeader).transactions().update(uri, transaction).execute();
-        } catch (URIValidationException | IOException e) {
+            var uri = "/private/transactions/" + transaction.getId();
```

Comment:
Could this prefix be a constant?

Reply:
as its only used once I would argue that would decrease the readability of the code